### PR TITLE
feat/add lowest_total_score and test

### PR DESCRIPTION
**Pull request by ChrisLyman**
adds lowest_total_score and test.  Just puggy backed on Becca's highest total score method.  Just used .map this time

### CODE REVIEW

**Comment by ChrisLyman** [comment]
created lowest_total_score method.  Piggeybacked on Becca's highest_total_score just used .map this time around.

**Review by rebeckahendricks — CHANGES_REQUESTED** [review]
I am curious on lines 25 and 26 (and 35 & 36) why the home_goals is accessing the :away_goals row and vice versa? Is that intentional?

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -22,10 +22,20 @@ def initialize(all_data_hash)
   def highest_total_score
     total_scores = []
```
Comment (mullinsand) [inline lib/stat_tracker.rb]: because we are now using map for this, we don't need the total_scores set as []

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -28,4 +28,14 @@ def highest_total_score
...
     total_scores.max
   end
+
+  def lowest_total_score
+    total_scores = []
+    @all_data_hash[:games].map do |row|
+      home_goals = row[:away_goals].to_i
+      away_goals = row[:home_goals].to_i
+      total_scores << home_goals + away_goals
+    end
```
Comment (mullinsand) [inline lib/stat_tracker.rb:37]: end.min

**Review by rebeckahendricks — APPROVED** [review]
This looks better!

**Review by mullinsand — APPROVED** [review]
New method looks sleek!

**Pull request merged**